pennylane_lightning/core/simulators/lightning_qubit/bindings/LQubitBindings_nb.hpp

```diff
@@ -161,7 +321,106 @@ void registerBackendSpecificStateVectorMethods(PyClass &pyclass) {
  * @param pyclass Nanobind's measurements class to bind methods.
  */
 template <class StateVectorT, class PyClass>
-void registerBackendSpecificMeasurements(PyClass &) {} // pyclass
+void registerBackendSpecificMeasurements(PyClass &pyclass) {
+    using PrecisionT = typename StateVectorT::PrecisionT;
+    using ComplexT = typename StateVectorT::ComplexT;
+
+    using np_arr_c = nb::ndarray<std::complex<PrecisionT>, nb::c_contig>;
+    using SparseIndexT = std::size_t;
+    using np_arr_sparse_ind = nb::ndarray<SparseIndexT, nb::c_contig>;
```

Comment:
Minor nitpick, but since these don't have to actually be numpy arrays it might be better to name these something else (e.g. `ArrayT`)